### DISCUSSION TITLE
Limiting access to tasmota devices

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16618] [Q&A] [answered]
Let's say I don't want my kids to change any settings but I want them to have direct access to the devices main menu to control the items and to see the statuses. There doesn't seem to be any of that in the default build but I imagine there is something I could add into a custom build?

For example, most pages should be fine but I'd like to limit configuration access.

**Comment by barbudor** [comment, marked as answer]
https://tasmota.github.io/docs/Commands/#webserver

'webserver 1'
You can only revert to 'webserver 2' through mqtt

**Reply by dac2020** [reply]
Great. I figured some would have needed such an option. I saw it in that page but it wasn't clear what it was. 
Thank you.
